app/controllers/course/assessment/submissions_controller.rb

```diff
@@ -27,14 +27,26 @@ def edit
   end
 
   def update
+    if params[:submit_answer]
```

Comment:
But currently we have the use case to submit a single answer, but it's only for coding question and has a attempt limit.